packages/core/src/conversation/ConversationService/ConversationService.ts

```diff
@@ -477,4 +483,16 @@ export class ConversationService {
       return this.establishMLS1to1Conversation(groupId, selfUser, otherUserId);
     }
   };
+
+  public async handleMLSMessageAddEvent(event: ConversationMLSMessageAddEvent) {
+    return handleMLSMessageAdd({event, mlsService: this.mlsService});
+  }
```

Comment:
comparing to `proteusService.handleOtrMessageAddEvent` function call below, `handleMLSMessageAdd` function (module) is not called directly in MLSService as we will need to inject `ConversationService` there -we need it for recovery from epoch mismatch when decrypting a message - conversation service knows how to join with external proposal by fetching group info first with its id.